montecarlo/config/SR1_parameters.ini

```diff
@@ -0,0 +1,315 @@
+[pax]
+parent_configuration = "pmt_afterpulsing"  # (should be checked for SR1)
+look_for_config_in_runs_db = False
+
+
+[WaveformSimulator]
+
+# S2 electron drift and extraction
+diffusion_constant_liquid =           31.73 * cm**2 / s  # Cathode at -8 kV, xenon:xenon1t:yuehuan:analysis:1sciencerun_s2width
```

Comment:
Thanks @jpienaar13 this would help a lot, I missed a lot of differences in my fax production that listed in those two configs.
@jpienaar13 Just one thing is that I have doubt on diffusion_constant here. In lax there is an EffectivePar = 0.925, I don't know how Yuehuan got 31.73 but without 0.925*31.73 = 29.35 it won't match with data.

Reply:
@zhut19 The diffusion constant is extracted from NG data in Figure A8 in @weiyuehuan's [note](https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenon1t:yuehuan:analysis:1sciencerun_s2width#appendix_2drift_velocity_and_diffusion_constant). I don't remember what the rational behind the introduction of the effective parameter was.

Reply:
I recently asked Yuehuan about this and he suggested to use the 31.73 for now - I guess the effective parameter is just to parametrize our ignorance, but we can't say whether the discrepancy comes from diffusion or drift velocy without some additional study.

Reply:
I still thinks that the diffusion constant should be set to 0.925*31.73 = 29.35 because it matches data, it works for both using drift time and using interaction depth. While without it the model slightly deviates from center of the width distribution.
<img width="1051" alt="pr_sep28" src="https://user-images.githubusercontent.com/16232525/30980478-360ece92-a44f-11e7-996b-59011ecf88fb.png">
@jhowl01 I believe the rational behind effective parameter is that the initial diffusion constant doesn't match data for different runs, and Yuehuan wants to keep the original diffusion constant fixed as we don't really understand what cause the change and instead use a scale factor to accommodate data.
@weiyuehuan Would you let us know what parameter you used for extracting 38.73 as diffusion constant? I am afraid that you might mixed up SR0 and SR1 drift velocities. the ratio between those two velocity is 1.44/1.335 = 1.0787 and two of your effective parameter 1.161 ~ 1.0787^2 while 0.925 ~ 1.0787^-1.

Reply:
@zhut19 @jhowl01 thanks for point out this. Yes, we don't understand why the effective parameters there, but need the value to adjust the format to the theory model as fun (3) in [Jelle's note](https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenon1t:aalbers:drift_and_diffusion). @zhut19 I used SR1 drift velocity in [FigA8](https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenon1t:yuehuan:analysis:1sciencerun_s2width#appendix_1explanation_of_tuning_parameters) for the fitting, but there is another free parameter w_0 in the fitting function,  so its hard to get precise diffusion constant. I just remember another note by [Chloé and Sara](https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenon1t_chloe_mcfield_parameters). They get the diffusion constant at 27.7 at SR1 drift field. Be noted there also free parameter in fun(7). @zhut19 if you find discrepancy between fax sim and our data, I think we need to adjust this value to match the data.

Reply:
This w_0 free parameter shouldn't prevent a good fit though, right? It is just a time-independent term in the physical model.

Thanks to both of you for explaining this more clearly. Still, going forward I think this effective parameter is confusing, especially because it's merged with the factor of 1.349 from 50p_area scaling, and with the factor of 2 from the diffusion model.

We should think about how to do this most rigorously - perhaps the best we could do is using the gate mesh position estimation as a gaussian prior for fitting the diffusion model to the width distribution we see? Knowing the SR1 diffusion constant is critical to simulation -> clustering -> processor acceptance of S2s. For now @weiyuehuan should we drop the 31.73 in favor of 29.35 as @zhut19 suggests?

Reply:
@jhowl01 @zhut19 I agree that we need more rigorously study about this. Since the 29.35 can match the data and fax better, we should use this updated value in fax.